pwiz_tools/Skyline/CommandArgs.cs

```diff
@@ -131,13 +131,16 @@ private static void SetCulture(string cultureName)
             (c, p) => c.HideAllChromatogramsGraph = true) {InternalUse = true};
         public static readonly Argument ARG_TEST_NOACG = new Argument(@"noacg",
             (c, p) => c.NoAllChromatogramsGraph = true) {InternalUse = true};
+        public static readonly Argument ARG_TEST_EXCEPTION = new Argument(@"exception",
```

Comment:
I don't see this getting used in a test. I think it should be. I guess you were just using it in your own manual testing?

Reply:
Good point. I was trying to write my test to provoke an exception without using this argument, but it is more readable to use this argument in the unit test.